google/cloud/bigtable/CMakeLists.txt

```diff
@@ -185,6 +185,7 @@ add_library(
     internal/legacy_row_reader_impl.h
     internal/logging_data_client.cc
     internal/logging_data_client.h
+    internal/mock_row_reader_impl.h
```

Comment:
That introduces testing code to the "production" library.  Please add to this library instead:

https://github.com/googleapis/google-cloud-cpp/blob/baa88cfb6a923db88c433cda0e76d612195baa04/google/cloud/bigtable/CMakeLists.txt#L263